@kiva/kv-shop/src/useBraintreeDropIn.ts

```diff
@@ -44,7 +45,14 @@ export interface DropInInitOptions {
 	paypalFlow?: PayPalFlowType,
 }
 
-export default function useBraintreeDropIn() {
+export interface DropInWrapper {
+	initDropIn: (options: DropInInitOptions) => Promise<Dropin>,
+	paymentMethodRequestable: Ref<boolean>,
+	requestPaymentMethod: () => Promise<PaymentMethodPayload | void>,
+	updateAmount: (amount: string|number) => void,
+}
+
+function initBraintreeDropin(): DropInWrapper {
```

Comment:
I can't comment on the line, but GitHub linting is saying that `event` is never used on line 95.

Reply:
We do have a `// eslint-disable-next-line no-unused-vars` comment on it, but maybe github doesn't evaluate those in-line linting rules?

Reply:
Not sure, haven't spent much time with GitHub's linter.